app/views/shared/_header_board.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="table-box">
+  <div class="table-img">
+    <img src="" alt="">
```

Comment:
sao lại thẻ rỗng thế

Reply:
the nay ve sau em moi xuat anh nguoi dung ra a, em sua thanh TO DO roi